stable/traefik/templates/configmap.yaml

```diff
@@ -30,6 +39,15 @@ data:
       {{- if .Values.ssl.enabled }}
       [entryPoints.https]
       address = ":443"
+      {{- if .Values.proxyProtocol.enabled }}
+      [entryPoints.http.proxyProtocol]
+        trustedIPs = [
+          {{- range $idx, $element := .Values.proxyProtocol.trustedIPs }}
+              {{- if $idx }}, {{ end }}
+              {{- $element | quote }}
+          {{- end -}}
+          ]
+      {{- end }}
```

Comment:
Can this bit go into a helper or something since it's duplicated in two places?

Reply:
I'd love to, but I'm afraid I'm going to need some guidance with that.

Reply:
Have a look at `_helpers.tpl` for examples of some helpers that are already used in this chart.

Reply:
I did, there's only one in there and it does a string replace. I'm looking for an example of how to replace an entire block of code. I'll keep looking in the other charts (tomorrow, it's 1.30am here).